src/web-index/webIndex.ts

```diff
@@ -97,12 +97,23 @@ export function createWebIndex(ctx: Context): IWebIndexInterface {
         bundles: scriptTags.join('\n'),
         css: cssTags.join('\n'),
       };
-      let contents = replaceWebIndexStrings(readFile(opts.templatePath), scriptOpts);
+
+      let fileContents = readFile(opts.templatePath);
+
+      const pluginResponse = await ctx.ict.awaitForSync('before_webindex_write', {
+        filePath: opts.templatePath,
+        fileContents,
+        bundles,
```

Comment:
I think we need to add script tags and cssTags here too (or just a list of css paths and js paths), don't you think?

Reply:
I'll look into it, but aren't these part of the buldes?

Reply:
Added the missing scriptTags and cssTags below, also updated docs